Base/usr/share/man/man5/pp.md

```diff
@@ -0,0 +1,36 @@
+## Name
+
+pp - Pixel Paint application file format (.pp)
+
+## Description
+
+Pixel Paint files are the products of Pixel Paint application.
+They contain all sorts of drawing information that is used by Pixel Paint application. 
+
+Each file contains the following information:
+- width
+- height
+- layers (optional)
+  - width
+  - height
+  - name
+  - locationx
+  - locationy
+  - opacity_percent
+  - visible
+  - selected
+  - bitmap
+- guides (optional)
+  - offset
+  - orientation
```

Comment:
Either provide actual structural/encoding details here, or specify that this is only a rough overview.

Reply:
I'll add a note about a rough overview in the next patch, but eventually I'd like to provide more details.

Reply:
I don't know much about Pixel Paint at the moment. I'm going to take on #11862 soon. I could improve paint's documentation (at least add more details instead of leaving a "rough overview" note) after / in process of closing mentioned issue.